src/OptimismGovernorV5.sol

```diff
@@ -337,6 +299,18 @@ contract OptimismGovernorV5 is
         return 100_000;
     }
 
+    function _quorumReached(uint256 proposalId)
+        internal
+        view
+        virtual
+        override(GovernorCountingSimpleUpgradeableV2, GovernorUpgradeableV2)
+        returns (bool)
+    {
+        (uint256 againstVotes, uint256 forVotes, uint256 abstainVotes) = proposalVotes(proposalId);
+
+        return quorum(proposalSnapshot(proposalId)) <= againstVotes + forVotes + abstainVotes;
```

Comment:
Traditionally, the `quorum` for Governors is just `for + abstain` isn't it? Any reason to change that?

Reply:
I'm afraid I don't have the context for this. This logic was inherited from [OptimismGovernorV2](https://github.com/voteagora/optimism-gov/blob/main/src/OptimismGovernorV2.sol), I assume it was part of the initial spec